### PR TITLE
Added code for day 19, part 1 (2015)

### PR DESCRIPTION
**Pull request by jdosornio**
Added solution for day 19 part 1

### CODE REVIEW

**Comment by jdosornio** [comment]
[2015-day-19-part-1](https://trello.com/c/vOUw38mb/22-2015-day-19-part-1)

File: src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#![feature(pattern)]
```
Comment (kanekv) [inline src/lib.rs]: @jdosornio build is failing - we've decided to use stable rust for this project

File: tests/day_19.rs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            "H => OH".to_string(),
+            "O => HH".to_string(),
+            "".to_string(),
+            "HOHOHO".to_string()
+        ])
+    );
+}
+
+#[test]
+fn test_day_19_part_1() {
```
Comment (kanekv) [inline tests/day_19.rs]: is this test fast enough?

Comment (jdosornio) [reply tests/day_19.rs]: If you mean that it doesn't hit overall `cargo test` performance, then yes

File: src/lib.rs (hunk 1 above)
Comment (jdosornio) [reply src/lib.rs]: I need nightly rust for `match_indices` to work. That means I should use another method/function which is supported by stable rust?

**Pull request merged**